rebar.config

```diff
@@ -19,7 +19,14 @@
 {xref_checks,[undefined_function_calls,undefined_functions,locals_not_used,
               deprecated_function_calls,warnings_as_errors, deprecated_functions]}.
 
-{dialyzer, [{warnings, [unmatched_returns, error_handling, race_conditions]}]}.
+{dialyzer, [
+    {warnings, [unmatched_returns, error_handling, race_conditions]},
+    {plt_location, "/home/ayodele/Documents/work/emqx/"},
```

Comment:
maybe change it to `"."` ? 
if it has to be an abs path, please remove it from here and inject in `rebar.config.erl`